hr_timesheet_employee/models/account_analytic_line.py

```diff
@@ -0,0 +1,34 @@
+# -*- coding: utf-8 -*-
+# Copyright 2017 Savoir-faire Linux
+# License LGPL-3.0 or later (http://www.gnu.org/licenses/lgpl).
+
+from odoo import _, api, fields, models
+from odoo.exceptions import ValidationError
+
+
+class AccountAnalyticLine(models.Model):
+
+    _inherit = 'account.analytic.line'
+
+    employee_id = fields.Many2one('hr.employee', 'Employee')
+
+    @api.multi
+    def compute_timesheet_values(self):
+        super(AccountAnalyticLine, self).compute_timesheet_values()
+
+        for line in self:
+            if line.user_id:
```

Comment:
@yasmineelmrini if the user_id is not filled here, this should raise an exception.